content/operators-workshop/01-Operators.md

```diff
@@ -0,0 +1,25 @@
+# What is an Operator
+
+- An application specific controller that extends the Kubernetes API
+- Manages a CRD (Custom Resource Definition)
+- CRD stores state and the operator performs operational tasks
+- Pattern first proposed by CoreOS in November 2016
+
+---
+
+# Example Operators
+
+- Etcd
+- Prometheus
+- Elastic Search
+- Rook
+
+---
+
+# Giant Swarm Operators
+
+- azure-operator
+- aws-operator
+- kvm-operator
+- cert-operator
+- draughtsman-operator
```

Comment:
Idea is we'd talk about some common operators e.g. Rook and also our operators and how they interact with each other.